cmd/edit/addon/cmd.go

```diff
@@ -201,9 +158,13 @@ func run(cmd *cobra.Command, argv []string) {
 					os.Exit(1)
 				}
 			}
+
+			if len(input.Options) > 0 && helper.Contains(input.Options, val) {
+				r.Reporter.Errorf("Expected %v to match one of the options /%v/", val, param.Options())
+				os.Exit(1)
+			}
```

Comment:
This should probably be done in an input validator?

Reply:
We want to validate in case of interactive input an parameter passing. The user has a limited set of options that can be passed that the addon can handle.

Reply:
I searched in ocm-sdk-go addon_parameter_type. I didn't see a ValidateOptions function there unfortunately.

Reply:
I found AddOnParameterOptionList here -> https://github.com/openshift-online/ocm-sdk-go/blob/main/clustersmgmt/v1/add_on_parameter_option_type.go#L191 Potentially should we be using this instead of this -> https://github.com/openshift-online/ocm-sdk-go/blob/main/clustersmgmt/v1/add_on_parameter_type.go#L46 ?